modules/standard/Set.chpl

```diff
@@ -93,6 +93,15 @@ module Set {
     }
   }
 
+  // If we have "chpl__serialize", assume we have "chpl__deserialize".
+  pragma "no doc"
+  proc _isSerializable(type T) param {
+    use Reflection;
+    pragma "no init"
```

Comment:
Do we need this pragma so that it compiles? I am mostly curious because I wouldn't expect to see this pragma in a param function

Reply:
Yes otherwise if the type is e.g. non-nilable you will get errors about default initialization.